pepper-apis/src/main/java/org/broadinstitute/ddp/export/json/structured/PicklistQuestionRecord.java

```diff
@@ -17,14 +19,21 @@
     private List<String> selected = new ArrayList<>();
     @SerializedName("optionDetails")
     private List<Map<String, String>> optionDetails = new ArrayList<>();
+    @SerializedName("nestedOptions")
+    private Map<String, List<String>> nestedOptions = new HashMap<>();
 
     public PicklistQuestionRecord(String stableId, List<SelectedPicklistOption> answer) {
         super(QuestionType.PICKLIST, stableId);
         if (CollectionUtils.isNotEmpty(answer)) {
             List<SelectedPicklistOption> selected = new ArrayList<>(answer);
             selected.sort(Comparator.comparing(SelectedPicklistOption::getStableId));
             for (SelectedPicklistOption option : selected) {
-                this.selected.add(option.getStableId());
+                if (StringUtils.isNotBlank(option.getParentStableId())) {
```

Comment:
Nice! I like approach 3 of keeping the answer array the top-level selected options, while the nested options are separated out.